README.md

```diff
@@ -175,6 +175,17 @@ number and the deployment network configuration allows connections on the
 configured port. As noted above, the final caveat is that in non-Kube
 deployments, port 9091 is expected for automatic port-scanning target discovery.
 
+## ARCHIVING RECORDINGS
+
+`container-jfr` supports a concept of "archiving" recordings. This simply means
+taking a snapshot of a recording at a point in time and saving this snapshot to
```

Comment:
Is this different from the existing snapshot recording term? Maybe use a synonym for snapshot?

Reply:
Reworded it to avoid that potential confusion, since they are different.